test/unit/attributes.js

```diff
@@ -1633,3 +1633,20 @@ QUnit.test( "SVG class manipulation (gh-2199)", function( assert ) {
 		assert.ok( !elem.hasClass( "awesome" ), "SVG element (" + this + ") toggles the class off" );
 	} );
 } );
+
+QUnit.test( "non-lowercase boolean attribute getters should not crash", function( assert ) {
+	assert.expect( 3 );
+
+	jQuery.each( [
+		"Checked",
+		"requiRed",
+		"SELECTED"
+	], function() {
+	    try {
+			jQuery( "<div>" ).attr( this );
```

Comment:
Should this be a test that mixed-case attributes are correctly retrieved? This just ensures they don't throw but not that they yield correct results.